SocialCareCaseViewerApi.Tests/V1/Factories/ResponseFactoryTests.cs

```diff
@@ -82,6 +83,50 @@ public void CanMapCaseNoteToBsonDocument()
             result.Should().BeEquivalentTo(expectedDocument);
         }
 
+        [Test]
+        public void HistoricalCaseNotesToDomainReturnsCaseNoteAsAStringWhenNoteTypeIsNull()
+        {
+            var historicalCaseNote = TestHelpers.CreateCaseNote();
+            historicalCaseNote.NoteType = null;
+
+            var result = ResponseFactory.HistoricalCaseNotesToDomain(historicalCaseNote);
+
+            result.GetValue("form_name").AsString.Should().BeEquivalentTo("Case note");
+        }
+
+        [Test]
+        public void HistoricalCaseNotesToDomainReturnsFormNameWithoutBracketsAndWhitespaceWhenNoteTypeHasASCInBrackets()
+        {
+            var historicalCaseNote = TestHelpers.CreateCaseNote();
+            historicalCaseNote.NoteType = "Case Summary (ASC)";
```

Comment:
We could update the TestHelpers.CreateCaseNote to use Random.String2 for the note type but also accept the value as a parameter for when we want to explicitly set it.

Reply:
That would be good, however I'm not sure how to then set the expectation without having the test code the same as prod code. Any ideas how we could get around that? 🤔

Reply:
Ahhh never mind I got it, so we set the test helper to have the note types to select from without the brackets and then for these tests I can explicitly set it through a parameter.

Reply:
I don't think the test code will be doing the exact same as prod.

In this example we'd have

```
[Test]
public void HistoricalCaseNotesToDomainReturnsFormNameWithoutBracketsAndWhitespaceWhenNoteTypeHasASCInBrackets()
{
    var historicalCaseNote = TestHelpers.CreateCaseNote(noteType: "Case Summary (ASC)");

    var result = ResponseFactory.HistoricalCaseNotesToDomain(historicalCaseNote);

    result.GetValue("form_name").AsString.Should().BeEquivalentTo("Case Summary");
}
```

Fundamentally test is doing the same thing we just aren't mutating the data post TestHelper function.

Reply:
Ahh noicee I like that - will update thanks!

Reply:
Updated in https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/202/commits/7bdaa07e3d8b4c5c11f90a57f784a87ed601ccd2.